src/localization/vaccineStrings.json

```diff
@@ -14,6 +14,14 @@
   "gil": {},
   "tl": {},
   "la": {},
+  "my": {
```

Comment:
Hmm, not sure what's going on here! (see https://github.com/openmsupply/mobile/pull/2993/commits/605b417c82a1a13840ddddaa40f5fdeffe0597a5).